src/commcare_cloud/ansible/cleanup_pg_repack.yml

```diff
@@ -0,0 +1,30 @@
+- name: Cleanup pg_repack configuration
+  hosts: pgbouncer_a6
+  tasks:
+    - include_vars:
+        file: roles/postgresql_base/defaults/main.yml
+
+    - name: Check if pg_repack is installed
+      stat:
+        path: "{{ postgres_install_dir }}/bin/pg_repack"
+      register: pg_repack_stat
+
+    - name: Display if pg_repack is installed
+      debug:
+        msg: "pg_repack is installed"
+      when: pg_repack_stat.stat.exists == True
+
+    - name: Check if pg_repack cron exists
+      stat:
+        path: "/etc/cron.d/pg_repack_commcarehq_synclogs"
+      register: pg_repack_cron
+
+    - name: Display if pg_repack cron exists
+      debug:
+        msg: "pg_repack cron is configured"
+      when: pg_repack_cron.stat.exists == True
```

Comment:
Nit: Is this task necessary? The next _Remove pg_repack cron file_ task should tell you if it existed by its "changed" state.

Reply:
Nope was just useful for debugging. I removed the two tasks that are no longer needed.